lib/api/core/bootstrap/pluginIndexBootstrap.js

```diff
@@ -41,6 +40,7 @@ class PluginBootstrap extends SafeBootstrap {
   async _boostrapSequence (collections) {
     await this._createCollections(collections);
 
+    // @todo RestictedStorage job to add to the cache
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)